scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -36,7 +37,7 @@ def __init__(self):
         self.simplifier = simplify.GeojsonSimplifier()
         self.simple_geojsons = {}
 
-    def generate_simple_geojsons(self, verbose=True):
+    def generate_simple_geojsons(self, verbose=True, show=False):
```

Comment:
If you got logging working, there is no need for this parameter.  Logging works without output to the stdout or stderr unless the user specifies e wants output.  However, logging is produced in some obscurely named files for inspection if a run failed.

Reply:
Removed `verbose` param.